selectrum-prescient.el

```diff
@@ -174,9 +174,29 @@ buffer. It does not affect the default behavior (determined by
 (selectrum-prescient-create-and-bind-toggle-command fuzzy "f")
 (selectrum-prescient-create-and-bind-toggle-command initialism "i")
 (selectrum-prescient-create-and-bind-toggle-command literal "l")
+(selectrum-prescient-create-and-bind-toggle-command literal-prefix "P")
 (selectrum-prescient-create-and-bind-toggle-command prefix "p")
 (selectrum-prescient-create-and-bind-toggle-command regexp "r")
 
+(defun selectrum-prescient-toggle-char-fold ()
+  "Toggle character folding in the current Selectrum buffer.
+
+See the customizable variable `prescient-use-char-folding'."
+  (interactive)
+  ;; Make sure that this change is local only to the current Selectrum
+  ;; buffer.
+  (setq-local prescient-use-char-folding
```

Comment:
Would it be better to also check here for the `prescient-filter-method` here, bun the other hand maybe one would also want to toggle the char folding before switching to the literal method right?

Reply:
I think it makes sense to keep the toggling of character folding separate from the toggling of the filter methods. Although only 2 filter methods currently use character folding, that can expand in the future.

Reply:
Okay, sounds better to keep it separate then!